step-generation/src/getNextRobotStateAndWarnings/index.ts

```diff
@@ -302,6 +308,28 @@ function _getNextRobotStateAndWarningsSingleCommand(
     //  no state updates required
     case 'heaterShaker/waitForTemperature':
       break
+    case 'absorbanceReader/openLid':
+      forAbsorbanceReaderOpenLid(
+        command.params,
+        invariantContext,
+        robotStateAndWarnings
+      )
+      break
+    case 'absorbanceReader/read':
```

Comment:
i think this is no state change? since closeLid is also called whenever you call `absorbanceReaderCloseInitialize` 

```suggestion
    case 'absorbanceReader/read':
    break
```

Reply:
great call thanks!